.github/workflows/stable-merge-check.yml

```
@@ -0,0 +1,122 @@
+
+  # Job to remove label on new commits and check for label presence
+  integration-test-check:
+    name: Integration Test Verification
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request' || contains(github.event.comment.body, '/integration-tests-ok')
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.8)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)